Gopkg.lock

```diff
@@ -606,6 +606,7 @@
     "github.com/goadesign/goa/middleware",
     "github.com/goadesign/goa/middleware/security/jwt",
     "github.com/goadesign/goa/uuid",
+    "github.com/gojuno/minimock",
```

Comment:
Why this is changed? I guess you need to 'make cleanup' and then re-run 'make' again?

Reply:
yes, I believe this is a side-effect of the call to a `makefile` goal. If this is a problem, I can revert that file change.

Reply:
Just do `make clean & make` to see if it's gone :)